eslint.config.cjs

```diff
@@ -1,111 +1,125 @@
-    },
-    plugins: { '@typescript-eslint': ts, import: importPlugin, prettier: prettierPlugin },
-    settings: {
-      'import/resolver': {
-        typescript: { alwaysTryTypes: true, project: ['./tsconfig.json'] },
-        node: true,
+      parserOptions: { 
+        ecmaVersion: 'latest', 
+        sourceType: 'module',
+        ecmaFeatures: {
+          jsx: true
+        }
       },
+      globals: {
+        React: 'readonly',
+        window: 'readonly',
+        document: 'readonly',
+        global: 'readonly',
+        HTMLElement: 'readonly',
+      }
+    },
+    plugins: { 
+      '@typescript-eslint': ts, 
+      prettier: prettierPlugin,
+      'react-hooks': reactHooks,
+      'react-refresh': reactRefresh,
     },
     rules: {
       'prettier/prettier': 'error',
-      'import/no-unresolved': 'error',
       '@typescript-eslint/quotes': [
         'error',
         'single',
-        { avoidEscape: true, allowTemplateLiterals: false },
-      ],
-      'import/no-duplicates': 'error',
-      'import/extensions': ['error', 'ignorePackages', { ts: 'never', tsx: 'never' }],
-      'no-console': 'error',
-      'import/order': [
-        'error',
-        {
-          groups: [
-            'builtin',
-            'external',
-            'internal',
-            'parent',
-            'sibling',
-            'index',
-            'object',
-            'type',
-          ],
-          'newlines-between': 'always',
-          alphabetize: { order: 'asc', caseInsensitive: true },
+        { 
+          avoidEscape: true, 
+          allowTemplateLiterals: false 
         },
       ],
-
-      // Preferir reglas de TS sobre las base
+      'no-console': 'warn',
       'no-unused-vars': 'off',
-      // '@typescript-eslint/no-shadow': 'error', // activar si lo necesitás
-      // Ajustes comunes de Airbnb (opcionales)
-      // 'no-plusplus': 'off',
-      // 'no-underscore-dangle': 'off',
-      // 'no-param-reassign': ['error', { props: true, ignorePropertyModificationsFor: ['acc', 'e', 'ctx', 'req', 'res', 'state'] }],
+      '@typescript-eslint/no-unused-vars': [
+        'warn',
+        { 
+          argsIgnorePattern: '^_',
+          varsIgnorePattern: '^_',
+        }
+      ],
+      '@typescript-eslint/no-explicit-any': 'warn',
+      '@typescript-eslint/explicit-module-boundary-types': 'off',
+      '@typescript-eslint/no-non-null-assertion': 'warn',
+      'no-undef': 'off', // TypeScript handles this
+      
+      // React Hooks rules
+      'react-hooks/rules-of-hooks': 'error',
+      'react-hooks/exhaustive-deps': 'warn',
+      
+      // React Refresh
+      'react-refresh/only-export-components': [
+        'warn',
+        { allowConstantExport: true },
+      ],
+    },
+  },
+
+  // Storybook files - more lenient rules
+  {
+    files: ['**/*.stories.{ts,tsx}'],
+    rules: {
+      'no-console': 'off',
+      '@typescript-eslint/no-explicit-any': 'off',
```

Comment:
For Storybook files you turned off `@typescript-eslint/no-explicit-any`. This conflicts with the project rule stating `never` use `any` (English project rule). Tests/docs/storybook may be more lenient, but the project rule is explicit.

**Sugerencia:**
```typescript
Do not disable the `no-explicit-any` rule globally for stories. If absolutely necessary for specific stories, use inline comments to allow `any` only in those files or use stronger typing for story args. Example:

// eslint-disable-next-line @typescript-eslint/no-explicit-any
const hack: any = ...

But prefer typed args: declare a proper type for the story's args.
```